src/Boot/tests/BootloadManager/BootloadersTest.php

```diff
@@ -50,6 +50,34 @@ static function(Container $container, SampleBoot $boot) {
         ]), $bootloader->getClasses());
     }
 
+    public function testBootloadFromInstance(): void
+    {
+        $container = new Container();
+
+        $bootloader = $this->getBootloadManager($container);
+
+        $bootloader->bootload([
```

Comment:
Could you add an example with anonymous class?

Reply:
Added